python/paddle/fluid/layers/control_flow.py

```diff
@@ -928,6 +957,10 @@ def while_loop(cond, body, loop_vars, is_test=False, name=None):
     """
     while_loop is one of the control flows. Repeats while_loop `body` until `cond` returns False.
 
+    Notice:
+        Local variables defined in ``body`` cannot be obtained through ``fetch_list`` of ``Executor`` , variables should
+        be defined outside ``body`` and placed in ``loop_vars`` for looping, then thses variables can be obtained by ``fetch_list`` .
```

Comment:
"thses variables can be obtained" -> "these variables can be fetched".

Reply:
Done.